src/sentry/static/sentry/app/views/performance/vitalDetail/table.tsx

```diff
@@ -0,0 +1,419 @@
+import {TableColumn} from 'app/views/eventsV2/table/types';
+
+import {DisplayModes} from '../transactionSummary/charts';
+import {
+  TransactionFilterOptions,
+  transactionSummaryRouteWithQuery,
+} from '../transactionSummary/utils';
+
+import {
+  getVitalDetailTableStatusFunction,
+  vitalAbbreviations,
+  vitalNameFromLocation,
+} from './utils';
+
+const COLUMN_TITLES = ['Transaction', 'Project', 'Unique Users', 'Count'];
+
+const getTableColumnTitle = (index: number, vitalName: WebVital) => {
+  const abbrev = vitalAbbreviations[vitalName];
+  const titles = [
+    ...COLUMN_TITLES,
+    `${abbrev}(p50)`,
+    `${abbrev}(p75)`,
+    `${abbrev}(p95)`,
+    `${abbrev}(Status)`,
+  ];
+  return titles[index];
+};
+
+export function getProjectID(
+  eventData: EventData,
+  projects: Project[]
+): string | undefined {
+  const projectSlug = (eventData?.project as string) || undefined;
+
+  if (typeof projectSlug === undefined) {
+    return undefined;
+  }
+
+  const project = projects.find(currentProject => currentProject.slug === projectSlug);
+
+  if (!project) {
+    return undefined;
+  }
+
+  return project.id;
+}
+
+type Props = {
+  eventView: EventView;
+  organization: Organization;
+  location: Location;
+  setError: (msg: string | undefined) => void;
+  summaryConditions: string;
+
+  projects: Project[];
+};
+
+type State = {
+  widths: number[];
+};
+
+class Table extends React.Component<Props, State> {
+  state = {
+    widths: [],
+  };
+
+  handleCellAction = (column: TableColumn<keyof TableDataRow>) => {
+    return (action: Actions, value: React.ReactText) => {
+      const {eventView, location, organization} = this.props;
+
+      trackAnalyticsEvent({
+        eventKey: 'performance_views.overview.cellaction',
+        eventName: 'Performance Views: Cell Action Clicked',
+        organization_id: parseInt(organization.id, 10),
+        action,
+      });
+
+      const searchConditions = tokenizeSearch(eventView.query);
+
+      // remove any event.type queries since it is implied to apply to only transactions
+      searchConditions.removeTag('event.type');
+
+      updateQuery(searchConditions, action, column.name, value);
+
+      ReactRouter.browserHistory.push({
+        pathname: location.pathname,
+        query: {
+          ...location.query,
+          cursor: undefined,
+          query: stringifyQueryObject(searchConditions),
+        },
+      });
+    };
+  };
+
+  renderBodyCell(
+    tableData: TableData | null,
+    column: TableColumn<keyof TableDataRow>,
+    dataRow: TableDataRow,
+    vitalName: WebVital
+  ): React.ReactNode {
+    const {eventView, organization, projects, location, summaryConditions} = this.props;
+
+    if (!tableData || !tableData.meta) {
+      return dataRow[column.key];
+    }
+    const tableMeta = tableData.meta;
+
+    const field = String(column.key);
+
+    if (field === getVitalDetailTableStatusFunction(vitalName)) {
```

Comment:
Should this be defined in `fieldRenderers.tsx` instead?

Reply:
I thought about it, but didn't seem generic enough. I don't want to do this for every bool, for now at least it's specialized to this view.

Reply:
I believe it's possible to specialize this for just this function like how we do it for `user_misery(...)` like [here](https://github.com/getsentry/sentry/blob/bc9f3e9390cc647d6d386caf7338b0d0e6935207/src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx#L358).

Reply:
Hmm still not sure, I'd also have to bring in vitals to field renderers, because the field is variable.

Reply:
Really? I was thinking something along the lines of adding the `compare_numeric_aggregate` function in `fieldRenderer` and having that rendered as a pass/fail. If that doesn't work, just ignore me.

Reply:
I'll give this a shot in another PR with the other field renderer change